saas/models/saas_template.py

```diff
@@ -158,7 +172,9 @@ def _on_template_created(self):
     @job
     def _install_modules(self):
         self.ensure_one()
-        domain = safe_eval(self.template_id.template_modules_domain)
+        domain = []
+        for module in self.template_id.template_modules_domain:
+            domain.append(module.name)
```

Comment:
don't use name ``domain`` for this value. Also, avoid ``for-block`` when possible. E.g. ``template_modules = [m.name for m in self.template_id.template_module_ids``